.github/actions/ci-results/action.yml

```diff
@@ -21,3 +21,9 @@ runs:
         flags: java
         files: |
           code-coverage/target/site/jacoco-aggregate-all/jacoco.xml
+    - uses: codecov/codecov-action@v1
+      with:
+        flags: javascript
```

Comment:
why not put `java,javascript` into the action above?

Reply:
the tag is per upload. So all java changes would get both tags and all javascript changes would get both tags

Reply:
urgh